conf/distro/poky-sota-nonostree.conf

```diff
@@ -0,0 +1,19 @@
+require conf/distro/poky.conf
+
+DISTRO = "poky-sota-nonostree"
+DISTRO_NAME = "OTA-enabled Linux"
+DISTRO_VERSION = "1.0"
+DISTRO_CODENAME = "sota-non-ostree"
+
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+PREFERRED_RPROVIDER_virtual/network-configuration ??= "networkd-dhcp-conf"
+
+SOTA_HARDWARE_ID ??= "${MACHINE}-non-ostree"
+
+PACKAGECONFIG_pn-aktualizr = ""
+
+IMAGE_INSTALL_append += "aktualizr"
+IMAGE_INSTALL_append += "aktualizr-shared-prov-creds"
+IMAGE_INSTALL_append += "aktualizr-shared-prov"
+IMAGE_INSTALL_append += "aktualizr-hwid"
```

Comment:
`aktualizr-hwid` is included automatically and the provisioning recipes need to be configurable (and they include the `-creds` recipes automatically if appropriate). Hence you could make these four lines a one-liner: `IMAGE_INSTALL_append += " aktualizr ${SOTA_CLIENT_PROV} "

Reply:
This variable is not set since I do not use `sota` distro and sota.bbclass is not applied in this case

Reply:
I see your point. There might be a smart way to do this, but I suppose for now, specifying the prov recipe is okay. But `-hwid` and `-creds` are unnecessary; they will be included automatically by other recipes.